protected/humhub/modules/admin/messages/de/user.php

```diff
@@ -1,89 +1,89 @@
 <?php
```

Comment:
Can you please revert changes to this file? Could cause problems with the Translation Site